Added code missing from repo but present on NuGet

matdavies opened the pull request:
These changes were decompiled using dotpeek.  Chances are these changes were made on Jake's machine directly, then published direct to NuGet, but the code was never pushed to GitHub.  If ever SqlObjectHydrator needs to change again, the changes should be based on this branch

jrote1 commented:
Was there something specific missing, a lot of changes seem like moving of files etc. 

Sorry if forgot to push some changes 

Also could you exclude `.vs` folder

matdavies committed:
Added code missing from repo but present on NuGet

These changes were decompiled using dotpeek.  Chances are these changes were made on Jake's machine directly, then published direct to NuGet, but the code was never pushed to GitHub.  If ever SqlObjectHydrator needs to change again, the changes should be based on this branch

matdavies commented:
He's alive! :)  I think `SafeTypeMapsAdd` was added here?

Have removed the .vs folder, and actually, I didn't mean to create the PR into the main repo here - meant to create it into the LT fork, so feel free to close this off if you want.

jrote1 commented:
That I am haha

That's annoying that forget to push as all the TypeMaps are tested, so would have lost that test aswell
